android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -94,16 +95,10 @@
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.wallet_search, menu);
-        final MenuItem searchItem = menu.findItem(R.id.search);
```

Comment:
I suppose this block was removed accidentally?

Reply:
No, it's for https://github.com/brave/brave-browser/issues/19897.